server/api/user/user.model.js

```diff
@@ -247,7 +247,7 @@ UserSchema
   .virtual('listInfo')
   .get(function() {
     return {
-      '_id':this._id.toString('binary'),
+      '_id':this._id.toString(),
```

Comment:
Nice found. I applied the same changes to line 221, and line 203  it solved the errors of unreable id when getting profiles. Shall we apply the changes to all of those lines?

Reply:
sounds great!